### DISCUSSION TITLE
V2.1 Clean install problem

### DISCUSSION THREAD

**Discussion by elvirdz** [plausible/analytics#4140] [Self-Hosted Support]
I try today clean install 2.1 CE i got problem with Register
Create my Account dont work (Apache Reverseproxy)

what can i try to register new account

**Comment by elvirdz** [comment]
Ok its problem in 2.1CE version, i just installed 2.0 and there is no problem

**Comment by ruslandoga** [comment]
👋 @elvirdz 

What exactly is the problem? Also please share more information about your setup, compose file, proxy config, env vars, and logs (browser console, docker conpose, reverse proxy) from the period when the problem occurred.

**Reply by ruslandoga** [reply]
Also please make sure your reverse proxy allows websocket connections.

**Reply by elvirdz** [reply]
Please change Apache Reverse Proxy Setting

	ProxyPreserveHost On
	ProxyPass / http://localhost:8000/
	ProxyPassReverse / http://localhost:8000/

	ProxyPass ws://localhost:8000
	ProxyPassReverse ws://localhost:8000
	ProxyAddHeaders On

with 2.1 i can not send Mail over my Mailserver, with old config of 2.0 there is no problem

plausible-1            | 19:53:00.997 request_id=F9LTUIhZKFzeq5oAAAIh [error] Failed to send e-mail

**Reply by ruslandoga** [reply]
PR is welcome!

Re email, please try the new SMTP adapter (it's mentioned in the [release notes](https://github.com/plausible/analytics/discussions/4125) under "New SMTP client").

**Reply by elvirdz** [reply]
Thx :) i just change from 465 port to 587 and it works

**Reply by ruslandoga** [reply]
The old adapter has problems with STARTTLS. If sending emails works on the submission port 587, it might mean STARTTLS doesn't happen and the messages are being sent in plain text.

**Reply by ruslandoga** [reply]
I've opened a PR with your suggested changes: https://github.com/plausible/community-edition/pull/128

**Comment by elvirdz** [comment]
a2enmod proxy proxy_http proxy_ajp remoteip headers proxy_wstunnel

Reverseproxy works ... after installation on the page for Register 
i click on "register new account" nothing hapend page dont work
